### ISSUE TITLE
Add Statens servicecenter

### ISSUE THREAD

**Issue by quite** [02JanDal/osm-bjk#46]
Maybe this could be something to add? Except I didn't look at code/data sources yet, maybe you don't scrape web, and prefer GIS like data, or at least some API?

Forgot the URL: https://www.statenssc.se/besok-servicekontor

**Comment by 02JanDal** [comment]
Though structured data definitely is to be preferred I'm not on principle opposed to scraped data when no alternative is available!

**Comment by quite** [comment]
Yeah understandable. As web unfortunately tends to get redesign every so often, scraping always brings the possibility of having to redo the job...
